internal/uvm/scsi/backend.go

```diff
@@ -170,6 +170,13 @@ func mountRequest(controller, lun uint, path string, config *mountConfig, osType
 		ResourceType: guestresource.ResourceTypeMappedVirtualDisk,
 		RequestType:  guestrequest.RequestTypeAdd,
 	}
+	// This option is set only for cwcow scratch disk mount requests
+	// where we need to format the disk with refs.
+	// For refs the scratch disk size should > 30 GB.
```

Comment:
nit: IIRC, the limit was 40GB for ReFS and that's what we enforce in the snapshotter, can you update the comment to say 40GB?